modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -0,0 +1,98 @@
+= Add Behavioral Headers to Simulated API Calls
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+You can use behavioral headers to change the behavior of the mocking service.
+
+
+.Use behavioral headers where you would normally specify other types of header in the API Console
+image::apid-behavioral-headers.png[]
+
+
+MS2-Delay:: Indicates how many milliseconds the request should take to answer. You might want to use this header to simulate load on your server or latency in a network.
++
+You can specify a value for this header as an integer number or as a range of integer numbers. If you specify a range, the value must have the form `min-max`. For example, given the value `500-1000`, the request will take a random time between 500 and 1000 milliseconds.
```

Comment:
integers are numbers, so seems like you could just say "integer" Also "integer" is a noun.

Reply:
alternatively, you might say "whole number," but "integer" seems better

Reply:
Fixed.